contrib/jenkins/build.sh

```diff
@@ -38,7 +38,7 @@ done
   || error_exit '--project is a required parameter'
 
 if [[ "$(uname -s)" == "Linux" ]]; then
-  GIT_HEAD="$(git describe --tags --always --abbrev=7 --dirty)"
+  GIT_HEAD="$(git describe --always --abbrev=7 --dirty)"
```

Comment:
@simonleung8 is it worth using `git rev-parse --short HEAD` here instead of this command? just because it's simpler and a bit easier to understand.

same question for the `Makefile`